pom.xml

```diff
@@ -452,5 +452,10 @@
       <artifactId>javax.annotation-api</artifactId>
       <version>1.3.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.xerial</groupId>
+      <artifactId>sqlite-jdbc</artifactId>
```

Comment:
S3Proxy cannot take on a [native dependency](https://github.com/xerial/sqlite-jdbc#how-does-sqlitejdbc-work) which cannot be used in some environments.

Instead of storing configuration in a database, why not use a properties or JSON file?  You can send a `SIGHUP` to the S3Proxy process to force it to re-read the file, similar to other Unix daemons.

Reply:
In my case, i need a programmable  way  to control bucket access' add/delete,  and we need to have HA feature, so i use db and cache.
BTW, for your suggestion, i can add an YAML/JSON cod to implete  Interface: AccessSecretManager. how  do you think?